0000-plugins.md

```diff
@@ -0,0 +1,155 @@
+- Feature Name: plugins
+- Start Date: 2024-12-02
+- RFC PR: [LemmyNet/rfcs#0000](https://github.com/LemmyNet/rfcs/pull/0008)
+- Lemmy Issue: [LemmyNet/lemmy#0000](https://github.com/LemmyNet/lemmy/issues/3562)
+
+# Summary
+
+Plugins will make Lemmy more flexible, and allow implementing features which are too niche for merging into Lemmy core. It will allow Lemmy to focus on core features, while allowing outside developers to contribute new features without using Rust.
+
+# Motivation
+
+TODO
+
+# Guide-level explanation
+
+## Extism
+
+An open source framework to develop plugins for Rust projects. It supports numerous different languages by compiling them to webassembly which gets called from Rust.
```

Comment:
[Extism](https://extism.org/) is an open source framework...

then you can remove the link below.